apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -82,12 +85,16 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     'accentButtonBackground', // accent
     'accentButtonText', // white
     // INPUTS - 14
```

Comment:
Probably want to change the #s in these comments to reflect the new counts of each section & the size of each group starting on [line 33 in SemanticSlotsDetailsList.tsx](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/apps/theming-designer/src/components/SemanticSlotsDetailsList.tsx#L33) will have to be changed too so that the DetailsList can resize

Reply:
Thanks @aneeshack4, good call! I just updated it